blazar/tests/manager/test_service.py

```diff
@@ -122,6 +133,7 @@ def setUp(self):
                                         'send_lease_notification')
 
         cfg.CONF.set_override('plugins', ['dummy.vm.plugin'], group='manager')
+        cfg.CONF.set_override('os_region_name', 'regionOne')
```

Comment:
What did you need to add this for?

Reply:
We don't . I think I added this because I was trying out testing service and enforcement at the same time. However, since enforcement is mocked in the service case then this is now entirely unnecessary. I have removed.